lib/matplotlib/widgets.py

```diff
@@ -584,6 +587,29 @@ def set_val(self, val):
         if self.eventson:
             self._observers.process('changed', val)
 
+    def update_range(self, valmin, valstep, valmax):
```

Comment:
```suggestion
    def update_range(self, valmin=None, valmax=None, valstep=None):
```
Order rhymes with `range(start, stop, step)`.
Allowing None, allows to e.g. only update one limit e.g. `slider.update_range(valmax=10)`.

Reply:
Done